CONTRIBUTING.md

```diff
@@ -97,12 +99,12 @@ Here is an overview of the general process for contributing (working on a new fe
 
 We use [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1904891) to manage our work on features, chores and bugfixes.
   
-Work is defined in Pivotal Tracker.  Discussions take place there, in Slack, and in live conversations in scrums and pair programming (currently uses Google Hangouts). 
+Work is defined in Pivotal Tracker.  Discussions take place there, in Slack, and in live conversations in scrums and pair programming (currently we mostly use Google Hangouts). 
 
 ### Features
 Any feature should include Cucumber acceptance tests and RSpec tests where appropriate.
 
-We try to avoid view and controller specs, and focus purely on unit tests at the model and service level where possible.  
+We try to avoid view and controller specs, and focus purely on unit tests at the model and service level where possible.  However, there are times when the functionality is complex enough in a controller that it merits a unit test (RSpec).  Views should be thoroughly tested and exercised with feature (cucumber) tests.  
 
```

Comment:
"Generally we do not use unit-level tests for views.  But we do unit tests for situations that require those - for instance, 1) confirming that menu views are rendered consistent with our menu structure design, and 2) confirming the presence of hidden elements, HTML attributes, etc. that could not be easily tested (if at all) in a cucumber test."

Reply:
I've incorporated this.  I did keep the sentence about `Views should be thoroughly tested...`